submissions/2024-06-20-OpenFF-Torsion-Multiplicity-Optimization-Training-Coverage-Supplement-v1.0/README.md

```diff
@@ -0,0 +1,64 @@
+# OpenFF Torsion Multiplicity Optimization Training Coverage Supplement v1.0
+
+## Description
+An optimization data set created to improve the coverage of both existing Sage
+2.2.0 proper torsion parameters and new parameters added through the torsion
+multiplicity project. The molecules in this data set were partly selected from
+the ChEMBL 33 database and partly generated manually to match parameters not
+covered by ChEMBL.
+
+## General Information
+
+* Date: 2024-06-20
+* Class: OpenFF Optimization Dataset
+* Purpose: Improve proper torsion coverage in Sage
+* Name: OpenFF Torsion Multiplicity Optimization Training Coverage Supplement v1.0
+* Number of unique molecules: 37
+* Number of filtered molecules: 0
+* Number of conformers: 259
+* Number of conformers per molecule (min, mean, max): 1, 7.00, 10
```

Comment:
Thanks @ntBre, dataset LGTM! Could you please just double check the number of conformers -- the notebook shows the following output:

```
* Number of unique molecules: 37
* Number of filtered molecules: 0
* Number of conformers: 84
* Number of conformers per molecule (min, mean, max): 1, 2.27, 10
* Mean molecular weight: 187.31
* Max molecular weight: 489.48
* Charges: [0.0, 1.0, 2.0]
```

Reply:
I tried `grep canonical_isomeric_explicit_hydrogen_mapped_smiles dataset.json | wc -l` to see if that could be a quick-and-dirty shortcut, but that turned up a *third* answer -- 592 matches 😂

Reply:
I'm pretty sure the number in the readme is correct, but I must not have saved the notebook after running it the last time. I'll spin it up again tomorrow and answer this conclusively!

I added an explicit energy cutoff in the factory based on Matt's example notebook, and that raised the number of conformers substantially. I'm not really sure which one is preferable, so maybe this was a good question to bring up either way!

Reply:
Thanks Brent!

Hmm 0.1 A is quite small -- looks like the default in the toolkit in 1 A. One likely outcome here is it may wind up generating many similar conformers that then may collapse to the same structure during optimization. For molecules this small I wouldn't expect too many conformers -- maybe a threshold of 0.5 A would get more diversity, if fewer numbers.

Reply:
I increased the threshold to 0.5, reran the notebook, and updated the readme! 

I also looked into why grep reported a different number of conformers, and it looks like the `canonical_isomeric_explicit_hydrogen_mapped_smiles` can be found in the `extras` field, the `attributes` field, and the `identifiers` field, but some of the entries don't seem to have `attributes`, which is why it's not a clean multiple of the number of molecules/conformers.

Reply:
Ahh that explains it. Thanks so much for looking into it! Everything LGTM, merging now :-)